src/__tests__/store/ui/reducer.test.ts

```diff
@@ -0,0 +1,180 @@
+import { UiActionTypes, IUiState } from "../../../src/store/ui/types";
+import { uiReducer } from "../../../src/store/ui/reducer";
+
+const initialState: IUiState = {
+    loading: false,
+    progress: 0,
+    isDropdownOpen: false,
+    isKebabDropdownOpen: false,
+    isSidebarOpen: true,
+    sidebarActiveItem: "dashboard",
+    sidebarActiveGroup: "feeds_grp"
+};
+
+describe('ui reducer', () => {
+    it('should return the initial state', () => {
+        expect(uiReducer(undefined, {
+            type: null
+        })).toEqual(
+            initialState
+        )
+    })
+
+    it('should fetch request', () => {
+        const testState: IUiState = {
```

Comment:
this is not needed. you are just using testState.loading which you can just pass the boolean value t or f.